packages/blade/src/components/Checkbox/CheckboxIcon/CheckboxIcon.tsx

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React from 'react';
+import getIn from '../../../utils/getIn';
+import makeMotionTime from '../../../utils/makeMotionTime';
+import { useTheme } from '../../BladeProvider';
+import { Svg, Path } from '../../Icons/_Svg';
+import { CheckboxIconWrapper } from './CheckboxIconWrapper';
+import { Fade } from './Fade';
+
+const CheckedIcon = ({ color }: { color: string }) => {
```

Comment:
Had to create Checked/Indeterminate Icon as component since we do not have `colors.brand.gray.500` `colors.brand.gray.200` in the checkbox

Reply:
Shouldn't we add these tokens if there is a need here?

Reply:
Lets not get blocked on this. We can do it later and won't be a breaking change also.